restapi/src/main/java/io/stargate/web/service/WhereParser.java

```diff
@@ -126,7 +126,12 @@
             conditions.add(conditionToWhere(fieldName, op, true));
           } else {
             Object val = value.asText();
-            Column.ColumnType columnType = tableData.column(fieldName).type();
+            Column column = tableData.column(fieldName);
+            if (column == null) {
+              throw new RuntimeException(
```

Comment:
Might want to throw an IllegalArgument and then catch it above so we can return a 400.

Reply:
Yes, I pinpointed and changed that. 🙂 Btw, there are some other RuntimeExceptions that could be casted to IllegalArgumentException in WhereParser#parseWhere. Wdyt?